### PR TITLE
Add load_to_bq Make task

### PR DESCRIPTION
**Pull request by duncanjbrown**
approach:
- define targets for CSV table exports
- define targets for BQ schemas
- phony tasks to upload to Google Cloud Storage and import to BQ

Generate the CSVs, infer the schemas, push to GCS, run the import!

Much to improve:
- env vars for variables
- schemas which actually match the tables, esp for geospatial data
- schemas for local_authorities and regions, which schema inference tool currently chokes on due to field length
- platform-agnostic tools (Docker?) so we can run in CI :D

### CODE REVIEW

File: ddl/tables/create_schools.sql [hunk 1]
```diff
@@ -2,6 +2,7 @@ drop table if exists schools;
 
 create table schools (
 	urn integer primary key,
+	ukprn varchar(128),
```
Comment (peteryates) [inline ddl/tables/create_schools.sql]: Looking at the raw column data it looks like UKPRN could be an integer.

**Comment by duncanjbrown** [comment]
Thanks – amended!


From: Peter Yates ***@***.***>
Date: Thursday, 1 June 2023 at 14:16
To: DFE-Digital/gias-query-tool ***@***.***>
Cc: BROWN, Duncan7 ***@***.***>, Author ***@***.***>
Subject: Re: [DFE-Digital/gias-query-tool] [WIP] Add load_to_bq Make task (PR #15)

@peteryates commented on this pull request.

________________________________

In ddl/tables/create_schools.sql<https://github.com/DFE-Digital/gias-query-tool/pull/15#discussion_r1213139355>:

> @@ -2,6 +2,7 @@ drop table if exists schools;



 create table schools (

        urn integer primary key,

+       ukprn varchar(128),

Looking at the raw column data it looks like UKPRN could be an integer.

File: dml/import_schools.sql [hunk 2]
```diff
@@ -39,6 +40,7 @@ insert into schools (
 
 select
 	sr."URN"::integer,
+	sr."UKPRN",
```
Comment (peteryates) [inline dml/import_schools.sql]: This _might_ be required, feel free to ignore if it's not though:

```suggestion
	sr."UKPRN"::integer,
```

Comment (duncanjbrown) [reply dml/import_schools.sql]: I've added — the [website](https://www.ukrlp.co.uk/) states that the UKPRN is an integer always beginning with `1`, so no need to deal with stray `0`s at the start of the code

**Pull request merged**